demo/src/BaseAgent.ts

```diff
@@ -25,6 +25,8 @@ import {
 import { IndySdkAnonCredsRegistry, IndySdkModule, IndySdkSovDidResolver } from '@aries-framework/indy-sdk'
 import { IndyVdrAnonCredsRegistry, IndyVdrModule, IndyVdrSovDidResolver } from '@aries-framework/indy-vdr'
 import { agentDependencies, HttpInboundTransport } from '@aries-framework/node'
+import { anoncreds } from '@hyperledger/anoncreds-shared'
+import { indyVdr } from '@hyperledger/indy-vdr-shared'
```

Comment:
this should also be the nodejs packages. Have you tried to run the demo and made sure it worked?